### ISSUE TITLE
ssb-friends should be optional

### ISSUE THREAD

**Issue by gmarcos87** [ssbc/ssb-threads#2]
I want to use this plugin in an open context, where there is no friendly relationship but content generated and replicated locally by local network.
The problem is that ssb-friends is a hard dependency and ssb-friends has other dependencies.
Can we modify this plugin so that, in case ssb-friends is detected, it uses the blocking system and if it is not detected, it works freely?
I know how to do it and I can generate a pull-request, my question is if this functionality would be accepted or if I directly make a fork.

Thanks

**Comment by staltz** [comment]
Good point, yes I agree with that idea. :)

**Comment by dominictarr** [comment]
Yes, I would merge this change. Especially if there where other plugins
that could be used in this spot that we could link to.
